desktop/libs/notebook/src/notebook/connectors/trino_tests.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
+# Licensed to Cloudera, Inc. under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  Cloudera, Inc. licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest import TestCase, mock
+from unittest.mock import MagicMock, patch, Mock
+from useradmin.models import User
+
+from desktop.auth.backend import rewrite_user
+from desktop.lib.django_test_util import make_logged_in_client
+from notebook.connectors.trino import TrinoApi
+
+
+class TestTrinoApi(unittest.TestCase):
+
+  def setUp(self):
```

Comment:
If we need to set all this up only once and not after every testcase, you can try looking upto `setUpClass`

Reply:
Hmm, good one, We can use it in many test files. Looks like it is comparatively fast also.